client/src/components/pages/personalProfile/index.js

```diff
@@ -0,0 +1,176 @@
+              {
+                whitespace: true,
+                message: 'Delete the spaces!'
+              },
+              {
+                min: 3,
+                message: 'First Name must be 3 charcter at least!',
+              },
+              {
+                required: true,
+                message: 'Please inter your First Name!',
+              },
+            ],
+          })(<Input placeholder='Your First Name' />)}
+        </Form.Item>
+
+        <Form.Item label="Last Name" className='form--item' >
+          {getFieldDecorator('last', {
+            rules: [
+              {
+                type: 'string',
+                message: 'The Last Name should be string!',
+              },
+              {
+                whitespace: true,
+                message: 'Delete the spaces!'
+              },
+              {
+                min: 3,
+                message: 'Last Name must be 3 charcter at least!',
+              },
+              {
+                required: true,
+                message: 'Please inter your Last Name!',
+              },
+            ],
+          })(<Input placeholder='Your Last Name' />)}
+        </Form.Item>
+
+        <Form.Item label="Email" className='form--item' >
+          {getFieldDecorator('email', {
+            rules: [
+              {
+                type: 'email',
+                message: 'The input is not valid E-mail!',
+              },
+              {
+                required: true,
+                message: 'Please input your E-mail!',
+              },
+            ],
+          })(<Input placeholder='Email' />)}
+        </Form.Item>
+
+        <Form.Item label="Password" hasFeedback className='form--item' >
+          {getFieldDecorator('password', {
+            rules: [
+              {
+                required: true,
+                message: 'Please input your password!',
+              },
+              {
+                whitespace: true,
+                message: 'Delete the spaces!'
+              },
+              {
+                validator: this.validateToNextPassword,
+              },
+              {
+                min: 8,
+                message: 'Password must be 8 charcter at least!',
+              }
+            ],
+          })(<Input.Password placeholder='Password' />)}
+        </Form.Item>
+
+        <Form.Item label="Confirm Password" hasFeedback className='form--item' >
+          {getFieldDecorator('confirm', {
+            rules: [
+              {
+                required: true,
+                message: 'Please confirm your password!',
+              },
+              {
+                validator: this.compareToFirstPassword,
+              },
+            ],
+          })(<Input.Password onBlur={this.handleConfirmBlur} placeholder='Confirm Password' />)}
+        </Form.Item>
+        <div className='form-div'>
+          <Form.Item >
+            <Button type="submit" className='form--btn-save' onClick={() => undefined} >
```

Comment:
@AbdallahAmmar96 I know it's not really part of your PR, but in the future it might make `<Button>` easier to use if it treated onClick as an optional prop, so you don't have to pass `onClick={() => undefined}` when you are using it.